Formula/varnish.rb

```diff
@@ -5,7 +5,7 @@ class Varnish < Formula
   mirror "https://fossies.org/linux/www/varnish-7.0.0.tgz"
   sha256 "8c7a5c0b1f36bc70bcbc9a48830835249e895fb8951f0363110952148cbae087"
   license "BSD-2-Clause"
-  revision 1
+  revision 2
```

Comment:
```suggestion
  revision 1
```

Reply:
This line should be removed for version bumps, actually.